chart/values.yaml

```
@@ -350,6 +344,14 @@ defaultSettings:
   # -- Setting that allows Longhorn to detect node failure and immediately migrate affected RWX volumes.
   rwxVolumeFastFailover: ~
 
+defaultBackupStoreSettings:
```

To prevent any confusion, remove `settings`
```suggestion
defaultBackupStore:
```

---

Add a comment for the purpose of defaultBackupStore`

---

Added. Thanks